src/code/z_actor.c

```diff
@@ -1464,7 +1467,7 @@ s32 func_8002F0C8(Actor* actor, Player* player, s32 flag) {
             dist = actor->xyzDistToLinkSq;
         }
 
-        return !func_8002F090(actor, D_80115FF8[actor->unk_1F].unk_4 * dist);
+        return !func_8002F090(actor, D_80115FF8[actor->unk_1F].leashMod * dist);
```

Comment:
does mod make sense for a name here? seems more like a scale factor to me

Reply:
This should probably go in your actor PR rather than here.

Reply:
yea it probably makes more sense there, but since its already done here might as well. it doesnt affect external/actor code.
do you agree that `leashScale` or factor or something makes more sense?

Reply:
`leashScale` seems fine.